t/cmd/lfstest-count-tests.go

```diff
@@ -16,12 +16,33 @@ import (
 )
 
 var (
+	// countFile is the path to a file (relative to the $LFSTEST_DIR) who's
```

Comment:
The documentation in this file is great and makes everything really easy to understand! ✨